From 340de6f2093594765c2df1f91842bfe90a1404f1 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@fedoraproject.org>
Date: Mon, 29 Sep 2014 15:02:32 +0200
Subject: [PATCH] Modernize the configure.ac

Without it, Automake 1.13 and above will refuse it.

See the details for AM_INIT_AUTOMAKE at:
   http://www.gnu.org/software/automake/manual/automake.html#Public-Macros
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7e2d5e3..237419a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,10 +1,10 @@
-AC_INIT
+AC_INIT([libev], [4.19])
 
 orig_CFLAGS="$CFLAGS"
 
 AC_CONFIG_SRCDIR([ev_epoll.c])
 
-AM_INIT_AUTOMAKE(libev,4.19) dnl also update ev.h!
+AM_INIT_AUTOMAKE
 AC_CONFIG_HEADERS([config.h])
 AM_MAINTAINER_MODE
 
-- 
2.1.0
